executor/linux/build.go

```diff
@@ -369,6 +369,7 @@ func (c *client) AssembleBuild(ctx context.Context) error {
 
 		// assume no privileged images are in use
 		containsPrivilegedImages := false
+		var privImage string
```

Comment:
These privileged image check changes look like they are from a different story? I say open a separate PR for this.

Reply:
will do!